pkg/kubectl/cmd/apply_test.go

```diff
@@ -279,6 +281,64 @@ func TestApplyObject(t *testing.T) {
 	}
 }
 
+func TestApplyObjectOutput(t *testing.T) {
+	initTestErrorHandler(t)
+	nameRC, currentRC := readAndAnnotateReplicationController(t, filenameRC)
+	pathRC := "/namespaces/test/replicationcontrollers/" + nameRC
+
+	// Add some extra data to the post-patch object
+	postPatchObj := &unstructured.Unstructured{}
+	if err := json.Unmarshal(currentRC, &postPatchObj.Object); err != nil {
+		t.Fatal(err)
+	}
+	postPatchLabels := postPatchObj.GetLabels()
+	if postPatchLabels == nil {
+		postPatchLabels = map[string]string{}
+	}
+	postPatchLabels["post-patch"] = "value"
```

Comment:
Yes, but still a bit different: this test case can ensure the new added labels can be printed out, but  the related issue is the rc has a label with value a and then update the label with value b, we expected the value b would be print out?